subsys/bluetooth/audio/tbs_internal.h

```diff
@@ -286,6 +286,15 @@ struct bt_tbs_in_uri {
 } __packed;
 
 #if defined(CONFIG_BT_TBS_CLIENT)
+
+#if defined(CONFIG_BT_TBS_CLIENT_BEARER_LIST_CURRENT_CALLS)
+#define BT_TBS_CLIENT_INST_READ_BUF_SIZE (BT_ATT_MAX_ATTRIBUTE_LEN)
+#else
+/* Need only be the size of call state reads */
+#define BT_TBS_CLIENT_INST_READ_BUF_SIZE (CONFIG_BT_TBS_CLIENT_MAX_CALLS \
+			* sizeof(struct bt_tbs_client_call_state))
```

Comment:
Shouldn't this be `MIN(BT_ATT_MAX_ATTRIBUTE_LEN, (CONFIG_BT_TBS_CLIENT_MAX_CALLS * sizeof(struct bt_tbs_client_call_state)))`? Sending more data is limited by maximum MTU size allowed anyway.